torch/csrc/jit/tensorexpr/tensor.cpp

```diff
@@ -29,11 +29,11 @@ void TensorOperationNode::SplitWithTail(
       tail_var,
       &tail_expr_node);
   if (!tail_expr_node) {
-    *tail_op = TensorOperation::make(tail_expr_node);
+    *tail_op = new TensorOperation(tail_expr_node);
```

Comment:
I'm highly confused by this bit with tail_op and tail_expr_node.  (Not the fault of this PR, just the double-indirection ** caught my attention and made me realize this was doing something unusual, so great :-) ).

Reply:
Yeah, probably no harm in changing it to a single *, but I wanted to keep the change as mechanical as possible.